src/js/franklin-dashboard.js

```diff
@@ -1,81 +1,95 @@
 'use strict';
 
+/* Dependencies */
 import angular from "angular";
 import packageJson from "../../package.json";
-import {
-    LoginComponent
-}
-from "./login/login.controller";
-
 require('angular-ui-router');
```

Comment:
Yeah, I know, talked with Andrew about this because the Broswerify ones don't work with ES6 (they are some third party libraries), do you know a way of wrap them in ES6?